wrapper.sh

```diff
@@ -188,6 +252,18 @@ if [[ $create == true ]]; then
     echo "run compile on po4gitbook to create new lessson"
     po4gitbook/compile.sh  > /dev/null 2>&1
     echo lesson $repo created in locale/ja/$repo
+
+    #restore _locale directory (only translate English lessons)
+    for dir in `git submodule |  grep "^+"  | cut -d" " -f2`
+        do
+        if [ -d $dir ]
+            then
+            cd $dir
+            git reset --hard
```

Comment:
This will restore the repo to the git `HEAD` commit (this undeletes the `_locale` directory).